antenna/configlib.py

```diff
@@ -0,0 +1,323 @@
+"""This module contains the configuration infrastructure allowing for deriving
+configuration from a .ini file and the process environment.
+
+In order of precedence:
+
+1. settings overrides (usually only applies when running tests)
+2. process environment
+3. .ini file (specified by ANTENNA_INI environment variable)
+
+Example of usage::
+
+    from antenna.configlib import config
+
+    DEBUG = config('DEBUG', default=True, parser='bool')
```

Comment:
To answer your last question, that's a case where you supply your own custom parser. Or, you write:
```python
DATABASES = config('DATABASES', default='postgres://foo:bar@baz.com:5432')
import dj_database_url
DATABASES = dj_database_url.parse(DATABASES)
```